content/en/docs/concepts/extend-kubernetes/operator.md

```diff
@@ -119,6 +119,7 @@ Operator.
   you implement yourself
 * [Operator Framework](https://operatorframework.io)
 * [shell-operator](https://github.com/flant/shell-operator)
+* [Java Operator SDK](https://github.com/java-operator-sdk/java-operator-sdk)
```

Comment:
This list is alphabetically sorted. Please ensure you add the entry with appropriate ordering.

Reply:
JOSDK just got much higher in the list :) 
thx, should be fixed.